packages/playground/remote/src/lib/playground-mu-plugin/0-playground.php

```diff
@@ -1,7 +1,9 @@
 <?php
 
+require(__DIR__ . '/playground-includes/playground_logger.php');
```

Comment:
Let's put that logic directly in `0-playground.php`. This is intentionally kept as a single file – there's a few importers (at least GitHub and Snapshots) that move this file to `/tmp`, import the data, and then moves it back. I don't love that and would like to redesign it as a part of the work on the PHP Blueprint library, but it is what it is for now.

Reply:
Done [1d840ed](https://github.com/WordPress/wordpress-playground/pull/1035/commits/1d840ed57134daf4c338bf85439a591ab4512477)